### DISCUSSION TITLE
Noop workshop intro feel a bit rushed

### DISCUSSION THREAD

**Discussion by cognitiaclaeves** [noop-inc/documentation#115] [General]
Maybe because it was at the end of the day, but I found most aspects of the video too fast to follow. My experience went like this: I was told to check out noop, so I pulled up the link found the download button, and installed it. I wanted some idea of how I could demo it, and the video was right there.

When I started the video, however, the voice didn't sound quite right. The cadence wasn't consistent. It sped up in weird places. Overall it was distracting. Then I realized I wasn't following what was happening on the screen, either. It seemed that there was no time to get a sense of what the clicks were doing, without watching the video more than once. ... Which I couldn't bring myself to do, because the voice was too unsettling. Maybe I'm still a bit biased toward human speech patterns.

It looks intriguing. I'm looking forward to learning more about it. I even see a possibility to use noop for future projects. But not today.

**Comment by 1800joe** [comment]
Thanks for the feedback! We'll work on the pacing.
